modin/pandas/series.py

```diff
@@ -339,6 +339,27 @@ def __iter__(self):
         """
         return self._to_pandas().__iter__()
 
+    def __invert__(self):
+        """
+        Apply bitwise inverse to each element of the `Series`.
+
+        Returns
+        -------
+        Series
+            New Series containing bitwise inverse to each value.
+        """
+        return super(Series, self).__invert__(shape_hint="column")
```

Comment:
any series after it was constructed should have `shape_hint="column"` in its query compiler, wouldn't it be easier to simply carry the shape hint for each map-like operation at the qc level instead?

Reply:
We register map operator for both Series and DataFrame at the QC. The QC doesn't know which object the operator is applied to so we have to pass a hint.

Reply:
Have you considered using `_pandas_class` to be able to use `shape_hint` in the base class?

Reply:
Not sure I get your point. Could you elaborate on it a bit? Anyway, there have already been the places where we've used a similar approach as in this PR so I think your point deserves a separate issue.

Reply:
> We register map operator for both Series and DataFrame at the QC. The QC doesn't know which object the operator is applied to so we have to pass a hint.

I mean, if a certain QC represents a series it will remain representing a series after any map operation, so we can simply preserve the `._shape_hint` in the result:
```python
# pandas query compiler
def abs(...):
    mf_result = ...
    return self.__constructor__(mf_result, shape_hint=self._shape_hint)
```